src/blockchain/evm/EvmContracts.ts

```
@@ -0,0 +1,78 @@
+import type { JsonFragment } from '@ethersproject/abi';
+import { Contract } from 'ethers';
+import { UnsContractName } from 'uns';
+import UnsNetworkConfig from 'uns/uns-config.json';
+
+import { StaticJsonRpcProvider } from './EvmProvider';
+
+type Abi = ReadonlyArray<JsonFragment>;
+
+const getAbi = async <T extends string>(name: T): Promise<Abi> => {
```

maybe just throw instead of catch it? it's critical to have the correct file name?

---

We have some contracts missing in `uns` (ProxyAdmin, WhitelistedMinter, TwitterValidationOperator). So even if we throw here we would have to catch and ignore the error. I added a log here so we don't miss it, but I don't think it's worth throwing.